tests/fixtures.py

```diff
@@ -22,27 +25,24 @@
     "shared": {
         "a_valid_reference": VALID_REF,
         "an_invalid_reference": NON_REF,
-        "a_malformed_reference": MALFORMED_REF,
     },
     "Test_identifier": {"test_matches_fixture": {"identifier": IDENTIFIER}},
     "Test_displayName": {"test_matches_fixture": {"display_name": "My Asset Manager"}},
     "Test_info": {
         "test_matches_fixture": {
-            "info": {kField_EntityReferencesMatchPrefix: "my_asset_manager:///"}
+            "info": {constants.kField_EntityReferencesMatchPrefix: "my_asset_manager:///"}
         }
     },
     "Test_resolve": {
-        "shared": {
-            "a_reference_to_a_readable_entity": EXISTING_REF,
-            "a_set_of_valid_traits": {LocatableContentTrait.kId},
-            "a_reference_to_a_readonly_entity": EXISTING_REF,
-            "the_error_string_for_a_reference_to_a_readonly_entity": "Entities are read-only",
-            "a_reference_to_a_missing_entity": "my_asset_manager:///missing_entity",
-            "the_error_string_for_a_reference_to_a_missing_entity": (
-                "Entity 'my_asset_manager:///missing_entity' not found"
-            ),
-            "a_malformed_reference": MALFORMED_REF,
-            "the_error_string_for_a_malformed_reference": "Entity identifier is malformed",
-        }
+        "a_reference_to_a_readable_entity": EXISTING_REF,
+        "a_set_of_valid_traits": {LocatableContentTrait.kId},
+        "a_reference_to_a_readonly_entity": EXISTING_REF,
+        "the_error_string_for_a_reference_to_a_readonly_entity": "Entities are read-only",
+        "a_reference_to_a_missing_entity": "my_asset_manager:///missing_entity",
+        "the_error_string_for_a_reference_to_a_missing_entity": (
+            "Entity 'my_asset_manager:///missing_entity' not found"
+        ),
+        "a_malformed_reference": MALFORMED_REF,
+        "the_error_string_for_a_malformed_reference": "Entity identifier is malformed",
```

Comment:
[question] Is it intended that this causes the resolve test to be skipped? As there are now no fixtures? (`shared` is a special key that uses the keys for all tests, otherwise hey need to be grouped under the test case name ).

```
test_when_resolving_malformed_reference_then_malformed_reference_error_is_returned (openassetio.test.manager.apiComplianceSuite.Test_resolve) ... skipped 'Required fixtures not found: a_reference_to_a_readable_entity'
test_when_resolving_missing_reference_then_resolution_error_is_returned (openassetio.test.manager.apiComplianceSuite.Test_resolve) ... skipped 'Required fixtures not found: a_reference_to_a_readable_entity'
test_when_resolving_read_only_reference_for_write_then_access_error_is_returned (openassetio.test.manager.apiComplianceSuite.Test_resolve) ... skipped 'Required fixtures not found: a_reference_to_a_readable_entity'
test_when_resolving_write_only_reference_for_read_then_access_error_is_returned (openassetio.test.manager.apiComplianceSuite.Test_resolve) ... skipped 'Required fixtures not found: a_reference_to_a_readable_entity'
test_when_unknown_traits_then_returned_specification_is_empty (openassetio.test.manager.apiComplianceSuite.Test_resolve) ... skipped 'Required fixtures not found: a_reference_to_a_readable_entity'
test_when_valid_and_unknown_traits_then_returned_specification_only_has_valid_traits (openassetio.test.manager.apiComplianceSuite.Test_resolve) ... skipped 'Required fixtures not found: a_reference_to_a_readable_entity'
test_when_valid_traits_then_returned_specification_has_those_traits (openassetio.test.manager.apiComplianceSuite.Test_resolve) ... skipped 'Required fixtures not found: a_reference_to_a_readable_entity'
```

Reply:
No idea what caused me to remove the shared from here, re-added [801d7c6](https://github.com/elliotcmorris/Template-OpenAssetIO-Manager-Python/pull/3/commits/801d7c67ccf11bf3cc09d0244ef6fd9d9fc78e92)

As a side note, is there a better way to see what compliance suite tests are being run that you know of? Even capturing pytest output with `pytest -vv -s` all you get is those dots and `s`'s, you cant see which particular tests are being run.

Reply:
Thanks!

https://openassetio.github.io/OpenAssetIO/testing.html#testing_manager_plugins_cli